### PR TITLE
Add "Toggle Natural Scrolling" command

### PR DESCRIPTION
**Pull request by wileymarques**
## Description

Script Command to change natural trackpad/mouse scrolling setting.

The script reverts the setting value each time.

## Type of change

- [ ] New script command

## Screenshot

https://user-images.githubusercontent.com/1015148/116467698-5a3c2400-a846-11eb-96f7-d57f000c7c3f.mov

## Dependencies / Requirements

None.

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by fe9lix — COMMENTED** [review]
Hey @wileymarques – I'm curious what the use case for this script is? In what situations would you run the script?

**Comment by wileymarques** [comment]
I often switch from trackpad and mouse while using my Mac. Natural scrolling isn't so "natural" for me on a mouse, so I keep turning this feature on and off during the day.

I think this is a thing for people coming from Windows. Since I have some colleagues at work that suffer from the same "issue".

File: commands/system/toggle-natural-scrolling.applescript [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# @raycast.schemaVersion 1
+# @raycast.title Toggle Natural Scrolling
+# @raycast.mode silent
+# @raycast.packageName System
+
+# Optional parameters:
+# @raycast.icon 🖱
+# @raycast.author Wiley Marques
+# @raycast.authorURL https://twitter.com/wileymarques
+# @raycast.description Script Command to change natural trackpad/mouse scrolling setting. Reverting the setting value each time.
```
Comment (unnamedd) [inline commands/system/toggle-natural-scrolling.applescript:13]:
```suggestion
# @raycast.description Change natural trackpad/mouse scrolling setting. Reverting the setting value each time.
```

File: commands/system/toggle-natural-scrolling.applescript [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+tell application "System Preferences"
+  activate
+  set current pane to pane "com.apple.preference.trackpad"
+end tell
+
+delay 0.6
```
Comment (unnamedd) [inline commands/system/toggle-natural-scrolling.applescript:20]: Why do you need this delay? Isn't it giving you the access to the trackpad settings right after you activate it?

**Review by fe9lix — APPROVED** [review]
Thanks for the explanation!

File: commands/system/toggle-natural-scrolling.applescript [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+tell application "System Events"
+  tell process "System Preferences"
+    click radio button 2 of tab group 1 of window "Trackpad"
+    click checkbox 1 of tab group 1 of window "Trackpad"
```
Comment (unnamedd) [inline commands/system/toggle-natural-scrolling.applescript:25]: If Apple decides to reorder the elements, your Script Command won't work correctly, am I right?

**Review by unnamedd — CHANGES_REQUESTED** [review]

**Pull request merged**